libs/designsystem/src/lib/components/chip/chip.component.spec.ts

```diff
@@ -39,6 +40,13 @@ describe('ChipComponent', () => {
     });
   });
 
+  it('should have touch area with minimum size equal to fat finger size', () => {
+    expect(window.getComputedStyle(spectator.element, '::after').minHeight).toEqual(
+      fatFingerSize()
+    );
+    expect(window.getComputedStyle(spectator.element, '::after').minWidth).toEqual(fatFingerSize());
+  });
```

Comment:
See other comment regarding test (segmented-control)

Reply:
Fixed in 44d80b33f8d0b350bdf2f3518eb6013be1bd122b